sherpa/plot/__init__.py

```diff
@@ -116,11 +117,27 @@
            'IntervalProjection', 'IntervalUncertainty',
            'RegionProjection', 'RegionUncertainty',
            'TemporaryPlottingBackend',
+           'set_backend',
            )
 
 _stats_noerr = ('cash', 'cstat', 'leastsq', 'wstat')
 
 
+def set_backend(name):
```

Comment:
I was thinking about being able to use a string for `TemporaryPlottingBackend` so :+1:

Reply:
Also, I can see `set_backend` being useful for users in a jupyter notebook who want to make sure that their favourite JS-plotting backend [*] is selected.

[*] maybe it isn't "bokeh" ;-)

Reply:
Once this all works, it'll be almost trivial to have > 4 backends (pylab, bokeh, plotly, altair+vega). But I don't do that!